services/QuillLMS/client/app/bundles/Comprehension/tests/actions/session.test.ts

```diff
@@ -57,7 +57,7 @@ describe('Session actions', () => {
 
     it('makes a POST request to the feedback API', () => {
       const expectedRequest = {
-        url: `https://us-central1-comprehension-247816.cloudfunctions.net/comprehension-endpoint-go`,
+        url: process.env.GOLANG_FANOUT_URL,
```

Comment:
apparently, jest does not extract process.env from webpack.client.*.js, so the new config file is necessary

some internet discussion: https://stackoverflow.com/questions/48033841/test-process-env-with-jest